commands.h

```diff
@@ -250,6 +250,35 @@ namespace commands {
 				if (hasCC1101) {
 					cc1101::writeCCreg(reg, val);
 				}
+
+        if (reg == 18) {       // bwidth
```

Comment:
das sieht falsch eingerückt aus :)

Reply:
und auch als ob es an der falschen Stelle wäre, da in Zeile  249 bzw. 251 die Register schon geschrieben wurden.

Reply:
> das sieht falsch eingerückt aus :)
Du kannst Probleme haben :-) Da hat wohl notepad++ mal wieder die Tabs durch Leerzeichen ersetzt. Leider kann ich die Änderungen nicht direkt in der Arduino-IDE ausführen, da es eine andere Verzeichnisstruktur verlangt.

In Zeile 249-252 wird das Register geschrieben, das aktuell geschrieben wird. Abhängig davon, wenn es das Register für die Bandbreite ist (if (reg == 18)), müssen zwei weitere Register (FREND1 + FIFOTHR) evtl. je nach Wert der der Bandbreite geändert werden.

Reply:
Ich kopiere für travisCI die Dateien aus den libs Ordnern in den SIGNALduino Ordner.

Wenn ich das ganze mit der Arduino IDE verarbeiten wollte, dann würde ich es über symbolische Links realisieren. Dann sind nach jedem checkout die Dateien aktuell in dem Ordner in dem Du es compilierst. :)

Notepad++ nutze ich übrigens selbst auch, das sollte in der Regel keine Zeilenumbrüche verändern.

Reply:
Ich habe mir eine Batch geschrieben, um die Dateien zu verschieben.

Reply:
das geht natürlich auch :)